boards/arm/v2m_musca/v2m_musca.yaml

```diff
@@ -0,0 +1,7 @@
+identifier: v2m_musca
+name: ARM V2M MUSCA
+type: mcu
+arch: arm
+toolchain:
+  - gnuarmemb
```

Comment:
@galak do we need to update the toolchain support?

Reply:
I mean, i suppose the SDK is supporting/will support ARMv8-M

Reply:
I added xtools since that will be how we get v8m support.

Reply:
yeah we will have it in the sdk ng.  This needs to be replaced with ds5 and arm gcc.  I'll make that change.

Reply:
cool

Reply:
added in ds5 (external) line underneath xtools